src/main/scala/org/allenai/deep_qa/pipeline/NoPassageMC.scala

```diff
@@ -0,0 +1,59 @@
+package org.allenai.deep_qa.pipeline
+
+import com.mattg.pipeline.Step
+import com.mattg.util.FileUtil
+import com.mattg.util.JsonHelper
+import org.json4s._
+
+import org.allenai.deep_qa.data.DatasetReader
+
+import scala.sys.process.Process
+import scala.sys.process.ProcessLogger
+
+/**
+  * This Step is a SentenceProducer that reads a file with MCReadingComprehensionInstances
+  * and removes the passages from them, turning it into a file with the format of a
+  * QuestionAnswerInstance. It does this by simply dropping the first column.
+  * Expected file format is "[passage][tab][question][tab][choices][tab][label]" or .
+  * "[index][tab][passage][tab][question][tab][choices][tab][label]".
+  * The output file format is "[question][tab][choices][tab][label]" or
+  * "[index][tab][question][tab][choices][tab][label]".
+ */
+class NoPassageMC(
```

Comment:
If something is a `Step`, it's probably best for the class name to end in `Step`.